### PR TITLE
Add support for arm64

### PR DESCRIPTION
**Pull request by Jason-ZW**
Add support for arm64.

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 FROM alpine:3.4
```
Comment (niusmallnan) [inline package/Dockerfile]: We can use `alpine:3.7` here, and skip `amd64=alpine:3.7`

File: scripts/package [hunk 2]
```diff
@@ -13,6 +13,13 @@ TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
 
 cp ../bin/container-crontab .
-docker build -t ${REPO}/container-crontab:${TAG} .
 
-echo Built ${REPO}/container-crontab:${TAG}
+if dapper -d --build -f Dockerfile -- -t ${REPO}/container-crontab:${TAG} .; then
+    echo Built ${REPO}/container-crontab:${TAG}
```
Comment (niusmallnan) [inline scripts/package]: How can we distinguish the architectures(arm64/arm) via tag

Comment (Jason-ZW) [reply scripts/package]: Already fixed. Tks

File: package/Dockerfile (hunk 1 above)
Comment (Jason-ZW) [reply package/Dockerfile]: Already fixed.Tks

**Comment by niusmallnan** [comment]
LGTM

@cloudnautique Our RancherOS also needs container-crontab, we should add arm64 support.
When you have a chance, can you please review and merge this.

After that, we can push the docker image of arm64 every time.

File: Dockerfile.dapper [hunk 3]
```diff
@@ -1,25 +1,29 @@
 FROM ubuntu:16.04
-# FROM arm=armhf/ubuntu:16.04
+# FROM arm=armhf/ubuntu:16.04 arm64=arm64v8/ubuntu:16.04
 
-ARG DAPPER_HOST_ARCH=amd64
+ARG DAPPER_HOST_ARCH
```
Comment (cloudnautique) [inline Dockerfile.dapper]: Can we keep the default amd64?

File: Dockerfile.dapper [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
 RUN apt-get update && \
     apt-get install -y gcc ca-certificates git wget curl vim less file && \
     rm -f /bin/sh && ln -s /bin/bash /bin/sh
 
-ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
+ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
-RUN wget -O - https://storage.googleapis.com/golang/go1.8.3.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
+RUN wget -O - https://storage.googleapis.com/golang/go1.9.2.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
```
Comment (cloudnautique) [inline Dockerfile.dapper]: should we go to golang 1.9.4? Or 1.10.x? Other projects are moving to 1.10.

File: scripts/package [hunk 5]
```diff
@@ -13,6 +13,13 @@ TAG=${TAG:-${VERSION}${SUFFIX}}
 REPO=${REPO:-rancher}
 
 cp ../bin/container-crontab .
-docker build -t ${REPO}/container-crontab:${TAG} .
 
-echo Built ${REPO}/container-crontab:${TAG}
+if dapper -d --build -f Dockerfile -- -t ${REPO}/container-crontab:${TAG}${SUFFIX} .; then
+    echo Built ${REPO}/container-crontab:${TAG}
+    elif [ "$?" != "42" ]; then
```
Comment (cloudnautique) [inline scripts/package]: more style then anything, is this the right way to indent? They are at the same level as the `if dapper... `

File: Dockerfile.dapper (hunk 3 above)
Comment (Jason-ZW) [reply Dockerfile.dapper]: Already fixed as your suggest. Thanks

File: Dockerfile.dapper (hunk 4 above)
Comment (Jason-ZW) [reply Dockerfile.dapper]: Already fixed as your suggest. Thanks

File: scripts/package (hunk 5 above)
Comment (Jason-ZW) [reply scripts/package]: Already changed the indent. Am i right? Thanks a lot.

**Pull request merged**